recipe/patches/0002-fix-pyproject.patch

```diff
@@ -0,0 +1,57 @@
+From 26829c2f184f85260f5ab9238cb342933203d93d Mon Sep 17 00:00:00 2001
+From: Lorenzo Pirritano <lpirritano@anaconda.com>
+Date: Thu, 2 May 2024 17:39:54 +0200
+Subject: [PATCH] fix pyproject
+
+---
+ pyproject.toml | 19 +++++++++++------
+ 1 file changed, 12 insertions(+), 7 deletions(-)
+
+diff --git a/pyproject.toml b/pyproject.toml
+index f1ab033..577d510 100644
+--- a/pyproject.toml
++++ b/pyproject.toml
+@@ -37,7 +37,7 @@ classifiers = [
+ dynamic = ['version']
+ 
+ [tool.hatch.version]
+-path = 'src/snowflake/_legacy/__init__.py'
++path = 'snowflake/_legacy/__init__.py'
```

Comment:
We should report all of these changes to Snowflake. This package seems pretty broken.

Reply:
Noted, thank you

Reply:
As per SF, we should not add it. So we need to remove this part of the patch.